serveradmin/resources/templates/resources/index.html

```diff
@@ -87,7 +87,9 @@
             <tbody>
                 {% for obj in hosts %}
                 <tr>
-                    <td>{{ obj.hostname }}</td>
+                    <td>
+                        <a href="{% url 'graphite_graph_table' %}?hostname={{ obj.hostname }}">{{ obj.hostname }}</a>
```

Comment:
We should rather use `urlencode` too for url parts, or we might end up in broken urls:
```suggestion
                        <a href="{% url 'graphite_graph_table' %}?hostname={{ obj.hostname | urlencode }}">{{ obj.hostname }}</a>
```